appcues/src/main/java/com/appcues/debugger/screencapture/ScreenCaptureProcessor.kt

```diff
@@ -0,0 +1,147 @@
+package com.appcues.debugger.screencapture
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.text.format.DateFormat
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.children
+import com.appcues.AppcuesConfig
+import com.appcues.R
+import com.appcues.monitor.AppcuesActivityMonitor
+import com.appcues.ui.ElementSelector
+import com.appcues.util.ContextResources
+import com.appcues.util.ResultOf
+import com.appcues.util.ResultOf.Success
+import java.util.Date
+
+internal class ScreenCaptureProcessor(
```

Comment:
@andretortolano I've moved the details around generating and (soon to be) saving a screen capture into this `ScreenCaptureProcessor` to consolidate, if you'd like to take another look.